### PR TITLE
edac: introduce edac feature into lkvs

### PR DESCRIPTION
**Pull request by laifryiee**
Intel i10nm EDAC driver supports the Intel 10nm series server integrated memory controller. Introduce this feature into lkvs test suite and add 2 test cases.

### CODE REVIEW

File: BM/edac/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# EDAC
+
+EDAC (Error Detection and Correction) framework provids support for detecting and correcting memory errors on systems with ECC (Error-Correcting Code) memory or other error detection mechanism.
```
Comment (ysun) [inline BM/edac/README.md:3]: typo: provids.

Comment (ysun) [reply BM/edac/README.md:3]: Others in the patch look good to me.

**Review by ysun — CHANGES_REQUESTED** [review]

**Comment by laifryiee** [comment]
Thanks for review. Commit is force updated.

**Comment by ysun** [comment]
Hi LaiYi,

One coding style check issue need fix:

>>>> Taints found in file: BM/edac/intel_edac.sh
>>>> Hints:
    BM/edac/intel_edac.sh:3[8](https://github.com/intel/lkvs/actions/runs/9219553211/job/25370823112?pr=229#step:4:9) - Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?., refer to https://github.com/koalaman/shellcheck/wiki/SC2181
Error: Process completed with exit code 1.>>>> Taints found in file: BM/edac/intel_edac.sh
>>>> Hints:
    BM/edac/intel_edac.sh:3[8](https://github.com/intel/lkvs/actions/runs/9219553211/job/25370823112?pr=229#step:4:9) - Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?., refer to https://github.com/koalaman/shellcheck/wiki/SC2181
Error: Process completed with exit code 1.>>>> Taints found in file: BM/edac/intel_edac.sh
>>>> Hints:
    BM/edac/intel_edac.sh:3[8](https://github.com/intel/lkvs/actions/runs/9219553211/job/25370823112?pr=229#step:4:9) - Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?., refer to https://github.com/koalaman/shellcheck/wiki/SC2181
Error: Process completed with exit code 1.

**Comment by laifryiee** [comment]
> Hi LaiYi,
> 
> One coding style check issue need fix:
> 
> > > > > Taints found in file: BM/edac/intel_edac.sh
> > > > > Hints:
> > > > > BM/edac/intel_edac.sh:3[8](https://github.com/intel/lkvs/actions/runs/9219553211/job/25370823112?pr=229#step:4:9) - Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?., refer to https://github.com/koalaman/shellcheck/wiki/SC2181
> > > > > Error: Process completed with exit code 1.>>>> Taints found in file: BM/edac/intel_edac.sh
> > > > > Hints:
> > > > > BM/edac/intel_edac.sh:3[8](https://github.com/intel/lkvs/actions/runs/9219553211/job/25370823112?pr=229#step:4:9) - Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?., refer to https://github.com/koalaman/shellcheck/wiki/SC2181
> > > > > Error: Process completed with exit code 1.>>>> Taints found in file: BM/edac/intel_edac.sh
> > > > > Hints:
> > > > > BM/edac/intel_edac.sh:3[8](https://github.com/intel/lkvs/actions/runs/9219553211/job/25370823112?pr=229#step:4:9) - Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?., refer to https://github.com/koalaman/shellcheck/wiki/SC2181
> > > > > Error: Process completed with exit code 1.

Modified accoring to reference coding style.

**Pull request closed without merging**